DiffTests/PatchSortTests.swift

```diff
@@ -51,13 +51,12 @@ class PatchTests: XCTestCase {
             switch (element1, element2) {
             case (.insert(let at1), .insert(let at2)):
                 return at1 < at2
-            case (.insert, .delete):
+            case (.insert(_), .delete(_)):
```

Comment:
I think it is too, because it even considers this exhaustive:
```swift
switch (element1, element2) {
    case (.insert(_), .insert(_)): break
    case (.insert, .delete): break
    case (.delete(_), .insert(_)): break
    case (.delete(_), .delete(_)): break
}
```
only one case without `(_)`